sdk/storage/azure-storage-common/inc/azure/storage/common/file_io.hpp

```diff
@@ -20,11 +20,7 @@
 
 namespace Azure { namespace Storage { namespace Details {
 
-#if defined(AZ_PLATFORM_WINDOWS)
-  using FileHandle = HANDLE;
-#elif defined(AZ_PLATFORM_POSIX)
-  using FileHandle = int;
-#endif
+  using FileHandle = FILE*;
```

Comment:
If we don't use different types for different platforms, this typedef isn't necessary anymore